pombola/south_africa/templates/writeinpublic/committee-messages.html

```diff
@@ -5,6 +5,22 @@
 {% block content %}
 
 <h2>Messages sent to {{ committee.name }}</h2>
+{% for hist in committee.org_history_old.all %}
+  <span class="committee-history">
```

Comment:
looks like this is repeated?

please factor out this code to a template that you include https://docs.djangoproject.com/en/3.1/ref/templates/builtins/#include

if you need pass a different variable name for the committee name, see the syntax further down that section